### ISSUE TITLE
Recurrent enums will give a error on conversion

### ISSUE THREAD

**Issue by ffinfo** [godatadriven/pydantic-avro#25]
In avro it's possible to reuse the same data type on multiple locations in the schema. This is already supported for objects but not yet for enums. Currently this will give an error like this:

```
    raise NotImplementedError(f"Type {t} not supported yet")
NotImplementedError: Type <EnumName> not supported yet

```

**Comment by ffinfo** [comment]
Relates to #24

**Comment by ffinfo** [comment]
Fixed in #26
